crates/breez-sdk/core/src/realtime_sync/storage.rs

```diff
@@ -13,20 +13,22 @@ use tracing::{debug, error};
 
 use crate::{
     DepositInfo, EventEmitter, ListPaymentsRequest, Payment, PaymentDetails, PaymentMetadata,
-    SdkEvent, Storage, StorageError, UpdateDepositPayload,
+    SdkEvent, Storage, StorageError, UpdateDepositPayload, persist::UserSetting,
 };
 use tokio_with_wasm::alias as tokio;
 
 const INITIAL_SYNC_CACHE_KEY: &str = "sync_initial_complete";
 
 enum RecordType {
     PaymentMetadata,
+    DefaultUserSettingIsApplied,
```

Comment:
How about just name it `UserSetting`? The way we treat it internally can be kept as implementation details as we might change it in the future.